db/schema.rb

```diff
@@ -84,6 +84,9 @@
   end
 
   create_table "common_applications", force: :cascade do |t|
+    t.boolean "authorized_representative", default: false
```

Comment:
should we do this as an enum? right now it breaks the pattern of not stating defaults on anything but the navigator

Reply:
Thanks for catching this. I've converted the field to an enum in 440e6cd.